src/onboarding2/Configure/Configure.js

```diff
@@ -51,10 +56,10 @@ function Configure({
             overflow: hidden;
           `}
         >
-          {mode === 'select' && (
+          {mode === CONFIGURE_MODE_SELECT && (
```

Comment:
I'm starting to think, maybe we should take the template selection part out of configure?

It'd probably simplify a lot of the props we pass around if we split these two (selecting template vs. configuring selected), and I don't think we have any transitions that depend on these two being together.